doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,203 @@
+  as the **scchvc** user if a valid configuration has been specified in
+  the **scchvc** user account.
+
+* **scc-hypervisor-collector.timer(8)** triggers the
+  **scc-hypervisor-collector.service(8)** unit to be run on a daily
+  basis by default.
+
+# CONFIGURATION SETTINGS
+
+Configuration settings are specified in YAML format and must contain:
+
+* a list of hypervisor **backends** from which to retrieve relevant
+  details
+* a collection of **credentials** that will be used to upload the
+  collected details to the SUSE Customer Center.
+
+See **scc-hypervisor-collector(5)** for details about the possible
+configuration settings.
+
+## CONFIGURATION MANAGEMENT
+The configuration settings can be specified in a single YAML file,
+or as a set of YAML files whose content will be merged together in
+lexical sort order.
+
+The latter scheme allows the configuration settings to be split up
+into multiple files, e.g. credentials can be specified in one file,
+and the hypervisor backends in one or more files.
+
+When splitting the hypervisor backend details among multiple files,
+the **backends** lists in each file will be merged together to form
+a single combined list; exact duplicates will be ignored but partial
+duplicates will result in errors.
+
+## ACCESS AND OWNERSHIP
+For security reasons only the non-root user that is running the
+**scc-hypervisor-collector** command should be able to access the
+specified configuration files.
+
+## CONFIGURATION VALIDATION
+The **--check** option can be utilised to check if the specified
+configuration settings are valid, or will report any errors that
+it detected.
+
+## SUPPORTED HYPERVISORS
+
+The following hypervisor types are supported:
+
+* VMWare vCenter (type **VMware**)
+* Libvirt (type **Libvirt**)
+
+Each hypervisor type has specific configuration settings that must
+be provided to permit the relevant details to be retrieved; these
+settings are documented in **scc-hypervisor-collector(5)**.
+
+# EXIT CODES
+
+**scc-hypervisor-collector** sets the following exit codes:
+
+* 0:  Run completed successfully, or configuration settings
+      are valid if check mode (**--check**) was specified.
+* 1:  An error occurred.
+
+# IMPLEMENTATION
+
+**scc-hypervisor-collector(1)** is implemented in Python.
+It communicates with the SUSE Customer Center via a RESTful
+JSON API over HTTP using TLS encryption.
+
+# ENVIRONMENT
+
+**scc-hypervisor-collector(1)** respects the HTTP_PROXY environment
+variable.  See https://www.suse.com/support/kb/doc/?id=000017441 for
+more details on how to manually configure proxy usage.
+
+# FILES AND DIRECTORIES
+
+**~/.config/scc-hypervisor-collector/**
+: Default configuration directory containing YAML configuration files,
+  merged together in lexical sort order.
+
+**~/scc-hypervisor-collector.log**
+: Default log file which will be automatically rotated and compressed
+  if it gets too large.
```

Comment:
We may want to include information related to the permissions set on this log file.

Reply:
Fixed - added some details in this section, and in the security considerations section.